frontend/src/views/Abfrage.vue

```diff
@@ -597,6 +611,25 @@ export default class Abfrage extends Mixins(
     this.isDeleteDialogAbfragevarianteOpen = true;
   }
 
+  private async handleRelevantAbfragevariante(abfrageTreeItem: AbfrageTreeItem): Promise<void> {
+    this.selectedAbfragevariante = this.getSelectedAbfragevariante(abfrageTreeItem);
+    if (!_.isNil(this.selectedAbfragevariante.id)) {
+      await this.setAbfragevarianteRelevant(
+        this.abfrageWrapped.infrastrukturabfrage.id as string,
+        this.selectedAbfragevariante.id as string,
+        true
+      ).then((dto) => {
+        this.saveAbfrageInStore(new InfrastrukturabfrageModel(dto));
+        this.$store.dispatch("search/resetAbfrage");
+        Toaster.toast("Relevante Abfragevariante wurde erfolgreich geändert", Levels.SUCCESS);
```

Comment:
Text evtl. umbennnen. -> `Abfragevariante <DER_NAME> in Abfrage <DER_NAME> als relevant markiert.`